distribution/docs/src/main/resources/_contents/_formats/file-types-supported-contents.adoc

```diff
@@ -0,0 +1,1544 @@
+==== Supported File Formats
+
+Using the various input transformers, ${branding} supports ingest of the following MIME types.
+
+.[[supported_file_types]]Supported File Types
```

Comment:
I don't think we support full, comprehensive metadata extraction for *all* of these types, do we?